test/integration/model/nested_multiple_elements_error_world/model.sdf

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+<sdf version="1.8">
```

Comment:
I think this file can be a standalone file in `/test/sdf` since it represents a world. A `model.config` file in a directory is used for models.

Reply:
Done